src/app/features/notification/notification.service.ts

```diff
@@ -18,6 +18,8 @@ import firebase from "firebase/compat/app";
 export class NotificationService {
   private messaging: any = null;
   readonly FIREBASE_CLOUD_MESSAGING_URL = `https://fcm.googleapis.com/v1/projects/${environment.firebaseConfig.projectId}/messages:send`;
+  private readonly NOTIFICATION_TOKEN_COOKIE_NAME = "notification_token";
+  private readonly DEVICE_NOTIFICATION_API_URL = "/api/v1/notification/device";
```

Comment:
This backend API might not be deployed on the testing site yet. Tom had created a first draft of the notification API, and he shared the payload with me. Based on that, I have implemented the API accordingly.